DataHub/UserInterface/03_ETLExtensions.md

```diff
@@ -0,0 +1,1764 @@
+
+![Next promotion base price to float](../../Assets/Screenshots/DataHub/Settings/ETL/Extensions/NextPromotionBasePriceToFloat.png "[Next special base price to date time]")
+
+This extension is used to map the promotion price of a PIM price field attribute to a destination attribute. The PIM price field is Actindo specific and contains not only the simple price, but also the base price, scale prices, promotion prices, the tax class including the tax rate and supported currencies. Each of this data is extracted from the price attribute by the corresponding extension. In this case, the promotion price is mapped to the destination attribute. Further settings are defined in the configuration.
+
+### Possible data type mappings
+
+| Destination attribute data type  | Source attribute data type      |
+|----------------------------------|---------------------------------|
+| Float                            | Price field                     |
+
+### Configuration
+
+- *Country*     
+  Select the appropriate country in the drop-down list to apply the correct tax rate to the price. All available countries are displayed in the list.
+
+- *Currency*   
+  Select the appropriate currency in the drop-down list. Only those currencies that are defined as a supported currency in the PIM price field attribute configuration are mapped. in All available currencies are displayed in the list.
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Destination gross*   
+  Enable the toggle to map the gross price of the PIM price field attribute. Disable the toggle to map the net price.
+
+[comment]: <> (Is that right? why do I need the country?)
+
+
+
+## Next special base price to date time
+
+[comment]: <> (SpecialBasePriceToDateTimeExtension)
+
+![Next special base price to date time](../../Assets/Screenshots/DataHub/Settings/ETL/Extensions/NextSpecialBasePriceToDateTime.png "[Next special base price to date time]")
+
+This extension is used to map the promotion date of a PIM price field attribute to a destination attribute. The PIM price field is Actindo specific and contains not only the simple price, but also the base price, scale prices, promotion prices, the tax class including the tax rate and supported currencies. Each of this data is extracted from the price attribute by the corresponding extension. In this case, the date of the next promotion is mapped to the destination attribute. Further settings are defined in the configuration.
+
+### Possible data type mappings
+
+| Destination attribute data type  | Source attribute data type      |
+|----------------------------------|---------------------------------|
+| Date time                        | Price field                     |
+
+### Configuration
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Get from date*   
+  Enable the toggle to map the start date of the promotion. Otherwise the end date of the promotion is mapped to the destination attribute. By default, the toggle is disabled.
```

Comment:
```suggestion
  Enable the toggle to map the start date of the promotion. Otherwise, the end date of the promotion is mapped to the destination attribute. By default, the toggle is disabled.
```